src/main/java/com/getindata/connectors/http/internal/security/SecurityContext.java

```diff
@@ -24,6 +24,8 @@
 
 import lombok.extern.slf4j.Slf4j;
 
+import com.getindata.connectors.http.internal.utils.ConfigUtils;
```

Comment:
nit: use static import.

Reply:
I did not use static import on purpose here.

There is debate, whether static imports are "ok" or not, and the con argument is that  from code you dont see that this const or method is not defined in this class, and you need to scroll up to find the import etc. Still nothing that IDE can sort out, but when reading code on GitHub -> this information is missing.

So this is something I'm trying here now, although I'm still using static imports for `assertThat` and all that jazz for Unit tests.